### PR TITLE
newsletter 2019-06-14

### PR DESCRIPTION
**Pull request by ryan-shea**
📰 🗞 fresh and ready!

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@ryan-shea could you open this against stable ?

File: app/retail/emails.py [hunk 1]
```diff
@@ -949,22 +949,22 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-As summer formally begins, we're looking forward to the weather, the outdoors, and the memories to come. We're also looking forward to the next round of hackathons that we're planning. Fill out this Typeform to receive more information about the
-as we announce the next set of events. You can look forward to new sponsors, big bounties, and more opportunities to collaborate on new projects. <a href="https://gitcoin.typeform.com/to/j7CSbV">The form lives here.</a>
+Ethereum transactions are reaching new heights recently with the highest frequency of activity on the network in months. Need some help with your gas pricing? We're way ahead of you. Check out our helpful gas visualizer to assist with your
+work on the network. <a href="https://gitcoin.co/gas"It lives here.</a>
```
Comment (danlipert) [inline app/retail/emails.py]: Missing closing tag here

**Comment by danlipert** [comment]
cherry-picked onto stable

**Comment by thelostone-mc** [comment]
@danlipert 🤦‍♂

**Pull request closed without merging**